jekyll/_cci2/browser-testing.md

```diff
@@ -37,7 +37,7 @@ WebDriver can operate in two modes: local or remote. When run locally, your test
 
 If Selenium is not included in your primary docker image, install and run Selenium as shown below::
 
-```yml
+```yaml
```

Comment:
is this change also because `yml` is not a supported language code for PrismJS?

Reply:
both `yml` & `yaml` keys are supported but just wanted to make everything consistent: https://prismjs.com/#supported-languages